src/Commands/HTTPS/InfoCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\HTTPS;
+
+use Consolidation\OutputFormatters\StructuredData\PropertyList;
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+/**
+ * Class InfoCommand
+ * @package Pantheon\Terminus\Commands\HTTPS
+ */
+class InfoCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Retrieve HTTPS information about an environment
```

Comment:
The new doc standard is to say `Displays`... e.g. `Displays HTTPS information about an environment`